src/app/donation-start/donation-start.component.ts

```diff
@@ -688,7 +688,7 @@ export class DonationStartComponent implements AfterContentChecked, AfterContent
 
   captchaReturn(captchaResponse: string) {
     this.captchaCode = captchaResponse;
```

Comment:
@NoelLH Might be helpful to sanity check that `captchaResponse` is not falsey in case a dev makes a change in the future which breaks this functionality without realising.

Reply:
@alihejazi Surely we would notice that in testing as it would break all donation creates? If it's not given a string it will also break at type level before the method is even entered, so this check would never be reached except specifically in the case of a blank string invocation, which would be a pretty surprising error.